src/commands/kat-tests-install-tools.yaml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  kind-version:
+    type: string
+    default: "0.7.0"
+  helm-version:
+    type: string
+    default: "2.16.5"
+  kubectl-version:
+    type: string
+    default: "1.18.0"
+steps:
+  - restore_cache:
+      key: v2-python-cache-{{ checksum "test/kat/Pipfile.lock" }}
+  - run:
+      name: "download kat script"
+      command: |
+        curl -Lo /tmp/kube-app-testing.sh -q https://raw.githubusercontent.com/giantswarm/kube-app-testing/master/kube-app-testing.sh
+        chmod +x /tmp/kube-app-testing.sh
+  - run:
+      name: "create temporary bin dir /tmp/bin"
+      command: |
+        mkdir /tmp/bin || true
+  - restore_cache:
+      key: bin-{{ checksum "/tmp/kube-app-testing.sh" }}
+  - run:
+      name: "install kind"
+      command: |
+        if [[ ! -f /tmp/bin/kind ]]; then curl -Lo /tmp/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/v<< parameters.kind-version >>/kind-$(uname)-amd64" && chmod +x /tmp/bin/kind; fi
```

Comment:
It's ugly that we install kind in different places in different way. I think that should be shared by integration test and this.

Reply:
I created https://github.com/giantswarm/giantswarm/issues/10059 for this in the WG board.

Both jobs have install tools commands so yes I think we could align them more. We use go to install in the integration test job but I'm fine with installing from a release.